tests/testOpsimFieldSlicer.py

```diff
@@ -116,10 +118,9 @@ def tearDown(self):
     def testWarning(self):
         self.testslicer.setupSlicer(self.simData, self.fieldData)
         with warnings.catch_warnings(record=True) as w:
-            warnings.simplefilter('always')
             self.testslicer.setupSlicer(self.simData, self.fieldData)
-            assert len(w) == 1
-            assert "Re-setting up an OpsimFieldSlicer" in str(w[-1].message)
+        self.assertEqual(len(w), 1)
+        self.assertIn("Re-setting up an OpsimFieldSlicer", str(w[-1].message))
```

Comment:
You may actually be right.  I think I've been the one mis-using with statements in some of the other sims unit tests.  I'll restore the old indentation.